samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml

```diff
@@ -0,0 +1,163 @@
+
+
+
+          ||Component name||Component version||
+
+          {% for component in _.finding.vulnerability_finding_events  %}
+
+          |{{ component.get("vulnerable_component.short_name") }}|{{
+          component.get("vulnerable_component.version") }}|
+
+          {% endfor %}
+        executionId: "{{ execution().id }}"
+        connectionId: ""
+        fieldSetters: []
+        environmentUrl: https://umsaywsjuo.dev.apps.dynatracelabs.com
```

Comment:
Please don't hardcode environments but use expressions.

Reply:
Thank you, used an expression for the environmentUrl